src/index.ts

```diff
@@ -57,7 +57,7 @@ registerWelcomeRoute(router)
 registerErrorRoute(router)
 
 router.get("/", (req: Request, res: Response) => {
-  res.redirect(303, "welcome")
```

Comment:
I think this is on purpose so that the redirect to welcome is relative and not absolute

Reply:
I think its still relative, but prefixed with a base path. When using `BASE_PATH` env the `/welcome` route also moves to `BASE_PATH/welcome`, so this redirect shoots into nowhere, and I would ague, this is a inconsistent behavior compared to the other routes. Pls correct me if im wrong